src/App.vue

```diff
@@ -1,28 +1,27 @@
 <template>
   <div id="app">
```

Comment:
Consider using the BootstrapVue containers. So it'll be like this for easy formatting:
`<b-container>
    <b-row>
        <b-col>
`